common/static/common/js/spec/components/tabbed_view_spec.js

```diff
@@ -73,66 +71,97 @@
                    });
 
                    it('marks the active tab as selected using aria attributes', function () {
-                       expect(view.$('.nav-item[data-index=0]')).toHaveAttr('aria-expanded', 'true');
-                       expect(view.$('.nav-item[data-index=1]')).toHaveAttr('aria-expanded', 'false');
+                       expect(view.$('.nav-item[data-index=0]')).toHaveAttr({
+                           'aria-expanded': 'true',
+                           'aria-selected': 'true',
+                           'tabindex': '0'
+                       });
+                       expect(view.$('.nav-item[data-index=1]')).toHaveAttr({
+                           'aria-expanded': 'false',
+                           'aria-selected': 'false',
+                           'tabindex': '-1'
+                       });
                        view.$('.nav-item[data-index=1]').click();
-                       expect(view.$('.nav-item[data-index=0]')).toHaveAttr('aria-expanded', 'false');
-                       expect(view.$('.nav-item[data-index=1]')).toHaveAttr('aria-expanded', 'true');
+                       expect(view.$('.nav-item[data-index=0]')).toHaveAttr({
+                           'aria-expanded': 'false',
+                           'aria-selected': 'false',
+                           'tabindex': '-1'
+                       });
+                       expect(view.$('.nav-item[data-index=1]')).toHaveAttr({
+                           'aria-expanded': 'true',
+                           'aria-selected': 'true',
+                           'tabindex': '0'
+                       });
                    });
-
-                   it('sets focus for screen readers', function () {
-                       spyOn($.fn, 'focus');
-                       view.$('.nav-item[data-url="test-2"]').click();
-                       expect(view.$('.sr-is-focusable.test-2').focus).toHaveBeenCalled();
+                   
+                   it('works with keyboard navigation RIGHT and ENTER', function() {
+                       view.$('.nav-item[data-index=0]').focus();
```

Comment:
@bjacobel had a lot of trouble with focus events during the JQuery upgrade. He might have advice for you here.